### ISSUE TITLE
[Bookworm] Update PDDF platform module for Bookworm

### ISSUE THREAD

**Issue by saiarcot895** [sonic-net/sonic-buildimage#16739]
#### Description

In the bookworm branch, the PDDF platform module has been disabled from the Broadcom build as it needs changes for Bookworm. This needs to be updated for Bookworm.

#### Steps to reproduce the issue:
1.
2.
3.

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by adyeung** [comment]
ETA 11/15

**Comment by saiarcot895** [comment]
@adyeung @geans-pin #16954 updates the PDDF platform module, so no additional work is needed here, I think.

**Comment by adyeung** [comment]
Thanks @saiarcot895, will review #16954

**Comment by saiarcot895** [comment]
Fixed by #16954.
